src/docs/quick-start.rst

```diff
@@ -1,12 +1,182 @@
+===========
 Quick Start
 ===========
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac ligula et felis aliquam posuere. Etiam ac augue vel tortor porttitor placerat vitae ac libero. Nunc a efficitur lectus. Aliquam malesuada congue arcu quis luctus. Quisque pretium dolor orci, ut ullamcorper felis tincidunt eget. Duis auctor felis ut maximus facilisis. Nam eu feugiat tortor.
+This page will provide brief instructions to configure SSSD with
+FreeIPA, AD, and LDAP.
+
+Quick Start IPA
+***************
+
+Before starting, make sure you have the following information.
+
+- Administrator credentials *i.e. admin*
+- FreeIPA domain name *i.e. sssd.io*
+- FreeIPA server hostname *i.e. ipa1.sssd.io*
+- FreeIPA server IP *i.e. 1.2.3.4*
+
+Install the necessary packages, for RHEL and clones the package is named **ipa-client**, and for Fedora it’s **freeipa-client**.
+
+.. code-block:: text
+  :caption: For RHEL and clones 
+
+    yum install -y ipa-client 
+
+.. code-block:: text
+  :caption: For Fedora  
+
+    dnf install -y freeipa-client 
+
+Make sure DNS is pointing to your IPA server(s) by checking **/etc/resolv.conf** and it having it contain an entry with the IPA server IP, if it’s missing go ahead and add it, as the first entry.
+
+.. code-block:: text
+  :caption: /etc/resolv.conf 
+ 
+    search sssd.io
+    nameserver 1.2.3.4 
+    nameserver 1.1.1.1
+
+Now the ipa-client-install common will work. Run the command **ipa-client-install** command and follow the prompts asking for your domain and server and then a user that can join the domain, which will be the administrator user. 
```

Comment:
s/common/command/

I would also use backticks to surround inline text commands, such as ``ipa-client-install``, other non-command text emphasis can be done with **bold**